archeologies-et-sciences-de-lantiquite.csl

```diff
@@ -0,0 +1,700 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="fr-FR">
+  <info>
+    <title>Archéologies et Sciences de l’Antiquité</title>
```

Comment:
I don't have much of an opinion; I can see the case for both. Given that, I'd go with what automated tools like slugify do and go without.